api/src/pcapi/scripts/update_admin_venue_type_to_other/main.py

```diff
@@ -0,0 +1,48 @@
+"""
+Job console documentation here: https://www.notion.so/passcultureapp/Documentation-Job-Console-769beeacd5a146de9c97b6f8ee544276
+Assumed path to the script (copy-paste in github actions):
+
+https://github.com/pass-culture/pass-culture-main/blob/PC-36325-api-modifier-toutes-les-venues-qui-sont-lieu-administratif-en-autre/api/src/pcapi/scripts/update_admin_venue_type_to_other/main.py
+
+"""
+
+import argparse
+import logging
+
+from pcapi.app import app
+from pcapi.core import search
+from pcapi.core.offerers import models as offerers_models
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+
+def main(not_dry: bool) -> list[int]:
+    venue_query = db.session.query(offerers_models.Venue).filter(
+        offerers_models.Venue.venueTypeCode == offerers_models.VenueTypeCode.ADMINISTRATIVE.name
+    )
+    venue_ids = list(venue_query.with_entities(offerers_models.Venue.id).all())
```

Comment:
détail tu peux te passer de with_entities en metant directement la colonne dans le query
```suggestion
    venue_query = db.session.query(offerers_models.Venue.id).filter(
        offerers_models.Venue.venueTypeCode == offerers_models.VenueTypeCode.ADMINISTRATIVE.name
    )
    venue_ids = list(venue_query.all())
```

Reply:
bon à savoir, merci pour l'astuce